Inventory.cs

```diff
@@ -3,34 +3,42 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Xml.Linq;
 
 namespace SimpleInv
 {
     public class Inventory : IInventory
     {
+        private List<Product> _products;
+
+        public Inventory() {
+            _products = new List<Product>();
+        }
         public void AddProduct(Product product)
```

Comment:
I don't see any validation rules in the Inventory? what if we tried to add a product with negative price ?